### DISCUSSION TITLE
Question about validation

### DISCUSSION THREAD

**Discussion by unrue** [ultralytics/yolov5#13135] [General]
Using Yolov5, during Training, I have mAP 50-95  of  0.24429 best net:

```
` epoch,      train/box_loss,      train/obj_loss,      train/cls_loss,   metrics/precision,      metrics/recall,     metrics/mAP_0.5,metrics/mAP_0.5:0.95,        val/box_loss,        val/obj_loss,        val/cls_loss,               x/lr0,               x/lr1,               x/lr2
662,            0.025693,            0.033209,            0.024151,             0.51583,             0.34646,             0.33872,             0.24429,            0.031564,            0.031409,            0.043762,           0.0031284,           0.0031284,           0.0031284

```
`

But using val.py to evaluate best trained net, I get a poor 0.00983

`                 Class     Images  Instances          P          R      mAP50   mAP50-95: 100%|██████████| 6/6 [00:15<00:00,  2.61s/it]
                   all        168        501     0.0123     0.0789    0.00983    0.00663
`

How is that possible? Possible explanation? Thanks.

**Comment by Phoenix8215** [comment]
YOLO models often have score and IoU thresholds for filtering predictions during inference. If these thresholds are different between training and evaluation, it might lead to poor results during evaluation.:-)

**Reply by unrue** [reply]
It looks very very different. Quite strange.
